plugins/pipelines/create.js

```diff
@@ -72,10 +72,27 @@ module.exports = () => ({
                         })
                         // get the default collection for current user
                         .then(pipeline =>
-                            collectionFactory.get({
-                                userId: user.id,
-                                type: 'default'
+                            collectionFactory.list({
+                                params: {
+                                    userId: user.id,
+                                    type: 'default'
+                                }
                             })
+                                .then((collections) => {
+                                    const defaultCollection = collections[0];
```

Comment:
not sure if you need the `defaultCollection` variable anymore since it is only used in one place
but it's okay we can fix this later